src/precision.hpp

```diff
@@ -44,7 +44,8 @@ enum class Precision {
     MIXED,
     Q78,
     BIN,  // TODO remove BIN,Q78, DYNAMIC, CUSTOM, MIXED
-    PRECISION_END
+    PRECISION_END,
+    INVALID
```

Comment:
PRECISION_END should be the last code

Reply:
Do we really want INVALID precision? The InferenceTensor API allows creating only "valid" tensors @atobiszei

Reply:
Fixed.